contents/docs/product-analytics/cutting-costs.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Cutting costs
+sidebar: Docs
+showTitle: true
+---
+
+We aim to be significantly cheaper than our competitors. In addition to our [ridiculously cheap pricing](/pricing), below are tips to reduce your product analytics costs:
+
+## Use anonymous events
+
+PostHog captures two types of events: **anonymous** and **identified**. Under our current [pricing](/pricing), anonymous events can be up to 4x cheaper than identified ones (due to the cost of processing them), so it's recommended you only capture identified events when needed.
+
+> **Note:** See our docs on [person profiles](/docs/product-analytics/person-profiles) for more information on the differences between anonymous and identified events.
+
+Examples of scenarios where you want to capture identified events are:
+
+- Tracking logged-in users in B2B and B2C SaaS apps
+- Doing user segmented product analysis
+- Growth and marketing teams wanting to analyze the _complete_ conversion lifecycle
+
+On the other hand, there are many use cases that don't require person profiles and can use anonymous events, such as:
+
+- Tracking a marketing website
+- Content-focused sites
+- B2C apps where users don't sign up or log in
+- High volume API server events
+  
+### How to capture anonymous events
+
+import HowToCaptureAnonymousEvents from '../../docs/libraries/js/_snippets/how-to-capture-anonymous-events.mdx'
+
+<HowToCaptureAnonymousEvents />
+
+## Configure autocapture
+
+[Autocapture](/docs/product-analytics/autocapture) is a powerful feature that captures many events automatically. It can also capture more than you need. To reduce which events are captured, you can [set an allow or ignore list](/docs/product-analytics/autocapture#reducing-events-with-an-allow-and-ignorelist).
+
+Alternatively, you can [disable autocapture](/docs/product-analytics/autocapture#disabling-autocapture) completely.
+
+## Only call `identify()` once per session
+
+It's only necessary to [identify](/docs/product-analytics/identify) a user once per session. To prevent sending unnecessary events, check `posthog._isIdentified()` before calling `identify()`:
+
+```js
+if (!posthog._isIdentified()) {
+    posthog.identify('distinct_id') // Replace 'distinct_id' with your user's unique identifier
+}
+```
+
+## Only call `group()` once per session
+
+In client-side SDKs, it's only necessary to call [`group()`](/docs/product-analytics/group-analytics) once per session. Prevent calling it multiple times to send fewer events and reduce costs.
+
+## Disable pageview or pageleave events
+
+PostHog automatically captures pageviews and pageleaves. This is great for product analytics, but it may mean sending more events than you need. An alternative is to disable automatically capturing these events and instead capture them manually for the pages you need.
```

Comment:
Simplifying as you mention automatically capture many times
```suggestion
PostHog automatically captures pageviews and pageleaves. This is great for web analytics, but it may capture more events than you need. An alternative is disabling these events and capturing them manually for the pages you need instead.
```